### DISCUSSION TITLE
欢迎加入编程世界的讨论组！

### DISCUSSION THREAD

**Discussion by neolee** [neolee/wop#107] [General]
## 👋 欢迎进入编程世界！

在这里你可以：

* 问你感兴趣的问题。
* 分享有趣的编程想法。
* 结识有共同爱好的伙伴。
* 保持开放心态、共同构建理想的编程学习社区 💪。

作为起步，可以在下面回帖介绍一下自己 😀

**Comment by CHANCYstone** [comment]
大家好！我是Chancy，大家寒假回去有没有想要学一些什么呀！？
